tests/rados/test_osd_async_recovery.py

```diff
@@ -0,0 +1,105 @@
+import random
```

Comment:
Please add the docstrings to the test case.

Reply:
Added the doc string.